pombola/writeinpublic/tests.py

```diff
@@ -14,7 +15,15 @@
 @requests_mock.Mocker()
 class ClientTest(TestCase):
     def setUp(self):
-        self.writeinpublic = client.WriteInPublic('https://example.com', 'test', '123', '42', 'https://example.net/p.json#person-{}')
+        adapter = Mock()
```

Comment:
Since you're creating this mock adapter before each test, could you store this as `self.adapter` and then make assertions about the calls you expect it to receive in each test below?